app/components/forms/wizard/basic-details-step.js

```diff
@@ -141,7 +141,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     $.fn.form.settings.rules.checkMaxMinOrder = () => {
       return parseInt($('.ui.form').form('get value', 'ticket_min_order'), 10) <= parseInt($('.ui.form').form('get value', 'ticket_max_order'), 10);
     };
-
+    $.fn.form.settings.rules.checkValidTimeDifference = () => {
```

Comment:
I am not sure but i think this can be done without using document. You can use ember for DOM. please try this.